compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -868,28 +904,31 @@ pub static BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
 
     rustc_attr!(
         rustc_promotable, Normal, template!(Word), WarnFollowing,
-        EncodeCrossCrate::No, IMPL_DETAIL),
+        EncodeCrossCrate::No, ),
     rustc_attr!(
         rustc_legacy_const_generics, Normal, template!(List: "N"), ErrorFollowing,
-        EncodeCrossCrate::Yes, INTERNAL_UNSTABLE
+        EncodeCrossCrate::Yes,
     ),
     // Do not const-check this function's body. It will always get replaced during CTFE.
     rustc_attr!(
         rustc_do_not_const_check, Normal, template!(Word), WarnFollowing,
-        EncodeCrossCrate::Yes, INTERNAL_UNSTABLE
+        EncodeCrossCrate::Yes, "`#[rustc_do_not_const_check] skips const-check for this function's body.\
+        It will always get replaced during CTFE",
```

Comment:
```suggestion
        EncodeCrossCrate::Yes, "`#[rustc_do_not_const_check] skips const-check for this function's body",
```

The second part doesn't necessarily apply to all functions that `#[rustc_do_not_const_check]` are used on. it's an odd documentation. I can followup later to change it.